projects/plugins/social/src/js/components/social-notes-toggle/index.tsx

```diff
@@ -58,6 +69,56 @@ const SocialNotesToggle: React.FC< SocialNotesToggleProps > = ( { disabled } ) =
 					'jetpack-social'
 				) }
 			</Text>
+			{ isEnabled && ! isUpdating ? (
+				<div className={ styles[ 'notes-options-wrapper' ] }>
+					<ToggleControl
+						label={ __( 'Append post link', 'jetpack-social' ) }
+						checked={ notesConfig.append_link ?? true }
+						className={ styles.toggle }
+						onChange={ onToggleNotesConfig( 'append_link' ) }
+						help={ __(
+							'Whether to append the post link when sharing the note.',
+							'jetpack-social'
+						) }
+					/>
+					{ notesConfig.append_link ? (
+						<>
+							<ToggleControl
+								label={ __( 'Use shortlink', 'jetpack-social' ) }
+								checked={ notesConfig.use_shortlink ?? false }
+								className={ styles.toggle }
+								onChange={ onToggleNotesConfig( 'use_shortlink' ) }
+								help={ __(
+									'Whether to use the shortlink instead of the full URL.',
+									'jetpack-social'
+								) }
+							/>
+							<ToggleControl
+								label={ __( 'Use parenthical link format', 'jetpack-social' ) }
+								checked={ notesConfig.use_parenthical_link ?? false }
+								className={ styles.toggle }
+								onChange={ onToggleNotesConfig( 'use_parenthical_link' ) }
+								help={
+									<span>
+										{ sprintf(
+											/* translators: 1 is the link format */
+											__(
+												'Whether to use the parenthetical link format like %1$s, instead of the standard link.',
+												'jetpack-social'
+											),
+											'(jetpack.com sn/12345)'
+										) }
+										&nbsp;
+										<ExternalLink href="https://jetpack.com/redirect/?source=jetpack-social-notes-help">
```

Comment:
We should create a separate task to update the documentation for the shortlink.

Reply:
Done ✅